scripts/linters/other_files_linter.py

```diff
@@ -237,6 +241,37 @@ def check_webpack_config_file(self):
         return concurrent_task_utils.TaskResult(
             name, failed, error_messages, error_messages)
 
+    def check_strict_ts_config(self):
+        """Checks if the files in strict TS config are sorted
+        alphabetically.
+
+        Returns:
+            TaskResult. A TaskResult object representing the result of the lint
+            check.
+        """
+        name = 'Sorted strict TS config'
+
+        failed = False
+        error_messages = []
+
+        strict_ts_config = json.load(
+            python_utils.open_file(STRICT_TS_CONFIG_FILEPATH, 'r'))
```

Comment:
`with python_utils.open_file(...)`

Reply:
Can't do it since in the spec file we are using `string_io` and it can't be used with the `with` statement.

Reply:
As per offline discussion, we finalized that we are going to have a `with open..` statement here.

Reply:
done